base/compiler/ssair/show.jl

```diff
@@ -31,11 +31,17 @@ function Base.show(io::IO, cfg::CFG)
     end
 end
 
-function print_stmt(io::IO, idx::Int, @nospecialize(stmt), used::BitSet, maxlength_idx::Int, color::Bool, show_type::Bool)
+function print_stmt(io::IO, idx::Int, @nospecialize(stmt), used::BitSet, maxlength_idx::Int, color::Bool, show_type::Bool,
+                    unstable_ssa::Union{Nothing, BitSet} = nothing)
```

Comment:
Why the `nothing` case instead of `BitSet()`, or requiring this parameter always?

Reply:
I think this was changed due to @fingolfin's comment https://github.com/JuliaLang/julia/pull/54501#discussion_r1612384199

Reply:
Hmm.. on second thought, I think this should probably not tie unstable-highlighting to `show_type` in the first place

I think we want, e.g., `π(%26)` and `return %10` to show the type-unstable SSA value in red:
![image](https://github.com/user-attachments/assets/f1dc14b0-a0af-4da0-93c9-0780a135b248)

Reply:
Maybe we need @fingolfin opinion to continue?